osu.Game/Screens/Play/Player.cs

```diff
@@ -248,7 +248,8 @@ private void initializeStoryboard(bool asyncLoad)
             storyboard = beatmap.Storyboard.CreateDrawable(Beatmap.Value);
             storyboard.Masking = true;
 
-            storyboardContainer.Add(asyncLoad ? new AsyncLoadWrapper(storyboard) { RelativeSizeAxes = Axes.Both } : (Drawable)storyboard);
+            if (asyncLoad) LoadComponentAsync(storyboard, s => storyboardContainer.Add(s));

```

Comment:
newlines required if you're using `else` with `if`.

same deal with second parameter – remove the variable